src/yunohost/app.py

```diff
@@ -714,11 +714,15 @@ def confirm_install(confirm):
         # extract "bar" and test if we know this app
         elif ('http://' in app) or ('https://' in app):
             app_name_to_test = app.strip("/").split("/")[-1].replace("_ynh", "")
+        elif '.' in app:
```

Comment:
What if app contains a path like ./myapp_ynh ? or even ../myapp_ynh ?

I think this checks should be done on the `app_id` obtained from the manifest, see around line 751...